src/elexmodel/models/BootstrapElectionModel.py

```diff
@@ -1054,13 +1061,64 @@ def _is_top_level_aggregate(self, aggregate: list) -> bool:
             len(aggregate) == 2 and "postal_code" in aggregate and "district" in aggregate
         )
 
+    def _format_called_contests_dictionary(self, called_contests: dict | None) -> dict:
+        """
+        Make sure that the called contest dictionary has the correct format
+        """
+        # called_contests is a dictionary where 1 means that the LHS party has won, 0 means that the RHS party has won
+        # and -1 means that the contest is not called. If called_contests is None, assume that all contests are not called.
+        if called_contests is None or len(called_contests) == 0:
+            called_contests = {i: -1 for i in range(self.n_contests)}
+
+        if len(called_contests) != self.n_contests:
+            raise BootstrapElectionModelException(
+                f"called_contests is of length {len(called_contests)} but there are {self.n_contests} contests"
+            )
+
+        called_contest_acceptable_values = {0, 1, -1}
+        if not all(
+            any(math.isclose(value, acceptable_value) for acceptable_value in called_contest_acceptable_values)
```

Comment:
What's the difference between using `math.isclose()` instead of `np.isclose()`? 🤔

Reply:
In that case, how would you feel about replacing it with `np.isclose()`?  We're already importing `numpy` and if there's no real difference, I don't see how it's worth the extra memory to import `math` for that one function 🤔

Reply:
done!